apps/backend/src/lib/odptApiPath.ts

```diff
@@ -24,3 +25,5 @@ type OmitApiKey<Path extends object> = {
 type OmitApiKeyFromPaths = OmitApiKey<paths>
 
 export type { OmitApiKeyFromPaths as paths }
+
+export type OdptClient = Client<OmitApiKeyFromPaths>
```

Comment:
`exprt { hoge, fuga }`でまとめたさあるかも？

```suggestion
type OdptClient = Client<OmitApiKeyFromPaths>

export type { OmitApiKeyFromPaths as paths, OdptClient }
```